src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -75,9 +75,11 @@ function ProviderSelectionField({
   useEffect(
     () => {
       if (showProvidersList) {
-        scrollAndFocus('h2');
-      } else if (mounted) {
+        scrollAndFocus('#providerSelectionHeader');
+      } else if (mounted && !providerSelected) {
```

Comment:
Don't think that you worked on this, but is `mounted` something we should be doing? Can we assume the component is always mounted if we reach a `useEffect()`?

Reply:
There might be a better name for that variable. Basically, it's used to try to skip some useEffect logic on the first render of the component.